src/service/member-api-adapter.js

```diff
@@ -67,6 +69,28 @@ module.exports = function (config) {
 
     var publicAPI = {
 
+        /**
+        * Retrieve details about all of the group memberships for one end user. The membership details are returned in an array, with one element (group record) for each group to which the end user belongs.
+        *
+        * In the membership array, each group record includes the group id, project id, account (team) id, and an array of members. However, only the user whose userId is included in the call is listed in the members array (regardless of whether there are other members in this group).
+        *
+        * **Example**
+        *
+        *       var ma = new F.service.Member({ token: 'user-or-project-access-token' });
+        *       ma.getGroupsByUser('42836d4b-5b61-4fe4-80eb-3136e956ee5c')
```

Comment:
not as of https://github.com/forio/epicenter-js-libs/blob/master/src/service/member-api-adapter.js
do you have something waiting to be merged?